api/TimeAddressableMediaStore.yaml

```diff
@@ -168,68 +188,169 @@ paths:
+    put:
+      summary: Update webhook details
+      description: |
+        Update the configuration of an existing webhook.
+
+        Webhook messages will conform to the
+        format in the `webhooks` section of the API docs, depending on the event type (as defined in the same section).
+        Availability of this endpoint is indicated by the name "webhooks" appearing in the `event_stream_mechanisms`
+        list on the service endpoint.
+
+        HTTP requests from the service SHOULD include a `api_key_name` header with the 'api_key_value' value. Clients SHOULD verify this against the value they provided when registering the webhook.
+
+        API implementations MAY partially support event filtering and transformations.
+        API implementations SHALL return a 400 response code if the filtering or transformation specified in the request is not supported.
+
+        API implementations SHOULD consider the security implementations of providing webhooks, and include appropriate mitigations against Server Side Request Forgery (SSRF) attacks and similar. API implementations SHOULD take appropriate steps to authorize the modification of existing webhooks. This may take the form of RBAC, or ABAC.
+      operationId: PUT_webhooks-webhookId
+      tags:
+        - Webhooks
+      requestBody:
+        content:
+          application/json:
+            example:
+              $ref: "examples/webhook-put.json"
+            schema:
+              $ref: schemas/webhook-put.json
+        required: true
+      responses:
+        "201":
+          description: Success. The webhook has been updated
+          content:
+            application/json:
+              example:
+                $ref: "examples/webhook-get-200.json"
+              schema:
+                $ref: schemas/webhook.json
+        "400":
+          description: Bad request. Invalid parameters or unsupported event filtering or transformation.
+        "404":
+          description: The requested Webhook ID in the path is invalid, or Webhooks are not supported by this API implementation
+    delete:
+      summary: Delete Webhook
+      description: |
+        Deletes the webhook. Availability of this endpoint is indicated by the name "webhooks" appearing in the `event_stream_mechanisms` list on the service endpoint.
+
+        API implementations SHOULD consider the security implementations of providing webhooks, and include appropriate mitigations against Server Side Request Forgery (SSRF) attacks and similar. API implementations SHOULD take appropriate steps to authorize the deleting of webhooks. This may take the form of RBAC, or ABAC.
+      operationId: DELETE_webhooks-webhookId
+      tags:
+        - Webhooks
+      responses:
+        "204":
+          description: No content. The webhook has been deleted.
+        "403":
+          description: Forbidden. You do not have permission to modify this webhook.
```

Comment:
Since we don't have a `read-only` field on the webhook, like we do on Flows i was just wondering if this 403 is needed explicitly? Since i guess this will be produced by the RBAC/ABAC/Coarse access rules? Unless i have missed something. Feels like if we are including a 403 here we should go back and add 403 to all methods?